src/python/pants/backend/python/register.py

```diff
@@ -15,17 +15,19 @@
 from pants.backend.python.targets.python_tests import PythonTests
 from pants.backend.python.tasks2.gather_sources import GatherSources
 from pants.backend.python.tasks2.pytest_run import PytestRun as PytestRun2
+from pants.backend.python.tasks2.python_binary_create import \
+  PythonBinaryCreate as PythonBinaryCreate2
```

Comment:
Indeed, but our isort cfg forces this to the escaped newlines version in this case (presumably it gives up on parens if it sees that it can't even fit a single import on the first line).

So until we change that, this has to remain as-is.